AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/other-information.model.ts

```diff
@@ -0,0 +1,22 @@
+export class OtherInformationModel {
+    constructor() {
+        this.judgeEmail = '';
+        this.judgePhone = '';
+        this.otherInformation = '';
+    }
+    judgeEmail?: string | undefined;
+    judgePhone?: string | undefined;
+    otherInformation?: string | undefined;
+
+    static init(otherInformation: string): OtherInformationModel {
+        let otherInfo: OtherInformationModel = {};
+        try {
+            otherInfo = JSON.parse(otherInformation);
+        } catch (error) {
+            otherInfo.judgeEmail = null;
```

Comment:
what's this for?

Reply:
judgeEmail and phone are nullable stored in the otherinformation model, which itself it stored as a json string prop in hearing